pkg/cmd/pipelinerun/list.go

```diff
@@ -60,7 +59,7 @@ tkn pr list -n foo",
 			prs, err := list(p, pipeline)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Failed to list pipelineruns from %s namespace \n", p.Namespace())
-				return err
```

Comment:
It should return `error`?

Reply:
If the resource is not found then we get the below error
Error: the server could not find the requested resource (get pipelineruns.tekton.dev)